resources/scripts.js

```diff
@@ -169,6 +169,27 @@ function gameOver() {
     betButtonState(false);
 }
 
+async function playerMoneyChange(change) {
+    playerMoneyElement.innerHTML = playerMoney;
+    if (change === 'subtract') {
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '75%';
+        playerMoneyElement.style.color = '#ff0000';
+        await new Promise(resolve => setTimeout(resolve, 500));
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '120%';
+        playerMoneyElement.style.color = '#00ff00';
```

Comment:
To be more clear, perhaps add a return here.

Reply:
Added returns where appropriate.